apps/web/pages/index.tsx

```diff
@@ -17,7 +17,6 @@ export default function Web() {
       </Head>
       {/* Put here already styled Home page component & remove Header! */}
       <Header variant="welcome" userName={user ? user?.username : 'Guest'} />
-      <Input variant="search" />
     </DynamicLayout>
   );
 }
```

Comment:
If I am not logged in, I should see the login or be at least redirected to the login page

Reply:
I fix that, thanks

Reply:
Done